pkgs/applications/video/video-trimmer/default.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  meson,
+  ninja,
+  pkg-config,
+  rustPlatform,
+  glib,
+  gtk4,
+  libadwaita,
+  blueprint-compiler,
+  gst_all_1,
+  ffmpeg-full,
+  wrapGAppsHook4,
+  appstream-glib,
+  desktop-file-utils,
```

Formatter says this is preferred:
```suggestion
{ lib
, stdenv
, fetchFromGitLab
, meson
, ninja
, pkg-config
, rustPlatform
, glib
, gtk4
, libadwaita
, blueprint-compiler
, gst_all_1
, ffmpeg-full
, wrapGAppsHook4
, appstream-glib
, desktop-file-utils
```